src/AzureClient/AzureClient.cs

```diff
@@ -1,24 +1,249 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
 #nullable enable
 
 using System;
 using System.Collections.Generic;
-using System.Text;
-using Microsoft.Jupyter.Core;
+using System.Linq;
+using System.IO;
 using System.Threading.Tasks;
-
+using Microsoft.Azure.Quantum.Client;
 using Microsoft.Identity.Client;
 using Microsoft.Identity.Client.Extensions.Msal;
-using System.Linq;
-using System.IO;
-using Microsoft.Quantum.Runtime;
+using Microsoft.Jupyter.Core;
+using Microsoft.Quantum.Simulation.Core;
 
 namespace Microsoft.Quantum.IQSharp.AzureClient
 {
     /// <inheritdoc/>
     public class AzureClient : IAzureClient
     {
+        private string ConnectionString { get; set; }
```

Comment:
```suggestion
        private string ConnectionString { get; set; } = string.Empty;
```

Reply:
It looks like the `#Pending` comment got edited into the suggestion somehow?

Reply:
ah. I'm using CodeFlow to read through these comments and it has no idea how to deal with suggestions.

---
In reply to: [427496055](https://github.com/microsoft/iqsharp/pull/144#discussion_r427496055) [](ancestors = 427496055)

Reply:
Ah, makes sense! I've always just used the built-in GitHub UI myself, but that makes a lot of sense.

Reply:
It does, but also, it's a good reason to stick with the GitHub UI. This isn't the only weird thing. CodeFlow is nice for reading code, but I may abandon it for commenting.

---
In reply to: [427498865](https://github.com/microsoft/iqsharp/pull/144#discussion_r427498865) [](ancestors = 427498865)